### DISCUSSION TITLE
Delete a package

### DISCUSSION THREAD

**Discussion by Louai99k** [community/community#54241] [Packages]
### Select Topic Area

Question

### Body

I've created a package using github registry and I want to delete it but the button is disabled. I really want to solve this asap.

**Comment by Louai99k** [comment]
I'm the owner and the creator of the package. I've deleted the repo of the package thinking that might solve this problem but it didn't. so now I have only the packages (they're two) but not their repos.

**Comment by airtower-luna** [comment]
Is the package public? The [documentation says](https://docs.github.com/en/packages/learn-github-packages/deleting-and-restoring-a-package#package-deletion-and-restoration-support-on-github):

> You cannot delete a public package if any version of the package has more than 5,000 downloads. In this scenario, contact [GitHub support](https://support.github.com/contact?tags=docs-packages) for further assistance.
